MaeParser.hpp

```diff
@@ -87,7 +88,7 @@ class EXPORT_MAEPARSER read_exception : public std::exception
         format(line_number, column, msg);
     }
 
-    virtual const char* what() const throw() { return m_msg; }
+    const char* what() const noexcept override { return m_msg; }
```

Comment:
yeah, I think you'll need to #iffdef around this. Or move the noexcept for now.

Reply:
Solved. I also removed a couple of spurious semicolons.